### PR TITLE
Added a notebook with a simple implementation of a Multinomial Naive Bayes Classifier

### LINKED ISSUE #52: Implement a Naive Bayes Classifier

### PR DESCRIPTION
**Pull request by adam-broussard**
Currently, when performing a 1:1 comparison (i.e., the top-predicted genre against the top-voted genre), this achieves ~51% accuracy.  When I compare this to a similar case with the raw TF-IDF classification we have been running with so far (with a similar simple 1:1 accuracy), I find ~25% accuracy!  This looks to be a big improvement!

### CODE REVIEW

**Comment by youngant** [comment]
Two problems I see:
1. When I run this notebook myself, I get ~47% accuracy instead of the ~51% you got. I'm not sure where this is coming from. Maybe something with the train-test split?
2. This brings up the other issue, which is that you're using the test data to do validation here. We need to split off validation data from the testing data if we're going to be comparing multiple methods.

**Comment by adam-broussard** [comment]
> 1. When I run this notebook myself, I get ~47% accuracy instead of the ~51% you got. I'm not sure where this is coming from. Maybe something with the train-test split?

I agree that this sounds like a slightly different random split - I'll have to check into it

> 2\. This brings up the other issue, which is that you're using the test data to do validation here. We need to split off validation data from the testing data if we're going to be comparing multiple methods.

Hmm, I didn't think about this as model validation, but maybe you are right, since we are using this as a metric to determine which model to use.  Maybe we should go ahead and split off a validation set (though the performance should, in practice, be pretty similar).

**Comment by adam-broussard** [comment]
I think it had just been awhile since I regenerated the training/test set - now I'm getting ~47% like you.

**Comment by adam-broussard** [comment]
I should also mention that after redoing the split, I find comparably lower accuracy from the TF-IDF method, so this isn't just degradation of the NBC.

**Comment by adam-broussard** [comment]
Now with more validation set!

**Comment by adam-broussard** [comment]
Now with more validation set!
